embeddings/pipeline/hugging_face_classification.py

```diff
@@ -30,16 +33,19 @@ def __init__(
         input_column_name: str,
         target_column_name: str,
         output_path: T_path,
+        sample_missing_splits: Tuple[float, float] = (0.1, 0.1),
```

Comment:
In my view sampling in pipelines should be optional instead of forcing sampling.

Reply:
Ok, I'll change that this an optional argument